readme.md

```diff
@@ -39,6 +39,13 @@ chrome_options.add_argument("--proxy-server={0}".format(proxy.proxy))
 browser = webdriver.Chrome(chrome_options = chrome_options)
 ```
 
+Before Running Tests
```

Comment:
Hmmm... I'm not sure what was going on there for me then. That was the only way I was able to get it to work. Maybe something wrong with my setup?